src/main/resources/anniv5Resources/localization/eng/coresetpack/Cardstrings.json

```diff
@@ -22,9 +22,7 @@
   },
   "${ModID}:RippedPecs": {
     "NAME": "Ripped Pecs",
-    "DESCRIPTION": "Gain 1 Strength for each Pack you've played a card from this combat. NL Exhaust.",
-    "UPGRADE_DESCRIPTION": "Gain 1 Strength for each Pack you've played a card from this combat.",
-    "EXTENDED_DESCRIPTION": [" NL (Gains {0} Strength.)"]
+    "DESCRIPTION": "Gain 1 Strength. NL ${ModID}:Synergy: Gain !M! Strength."
```

Comment:
Not sure if this was Gk's intent with the comment, but what came to mind for me was the version of this card that was just: `Synergy: Gain 3 (4) Strength`.

It's a bit weaker than `Gain 1 Strength. Gain 2 (3) Strength` version, but it should still be a strong card and I like it for a few reasons: (1) very simple text (2) uses the design space of a do-nothing-without-synergy card (3) pleasing symmetry with Spot Weakness.

Reply:
If we were to do that, I'd prefer it be "Requires Synergy".  Having a playable card as potent as that which will either be big, or do nothing, depending on if its glowing or not, is a pitfall for players to fall into.  Someone suggested it always gaining 1 Strength, which seemed more approachable.  My preferred would still remain 'Requires Synergy'.

Reply:
While the same argument can be made of Spot Weakness, you at least have to drag spot weakness onto a target, which has its intent directly listed over it.  It's harder to whiff a spot weakness than to whiff a self-targeted card that has a hand-based conditional.

Reply:
I would be completely fine with the Requires Synergy version.

I don't really feel that it's necessary to have Requires Synergy instead of just Synergy, but I understand how you can draw a distinction between this and Spot Weakness. And even with the extra word, I still like the Requires Synergy version more than the 1 Strength/Synergy: 2 (3) Strength version.

Really not something I feel that strongly about, but there's something in me that is pulled towards the version of this card that does nothing without Synergy. All the stuff I said in my first comment still applies, and it just feels better somehow.